### ISSUE TITLE
embark collect can't open any file that has'been searched not from the root directory.

### ISSUE THREAD

**Issue by ClassAteam** [doomemacs/doomemacs#7192]
### I confirm that...

- [X] I have searched the [issue tracker](https://github.com/doomemacs/doomemacs/issues), [documentation](https://docs.doomemacs.org), [FAQ](https://docs.doomemacs.org/-/faq), [Discourse](https://discourse.doomemacs.org), and [Google](https://google.com), in case this issue has already been reported/resolved.

- [X] I have read "[How to Debug Issues](https://doomemacs.org/d/how2debug)", and will use it to provide as much information about this issue as possible.

- [X] The issue can be reproduced on the **latest** available commit of Doom.

- [X] The issue can be reproduced on a stable release of Emacs, such as 27 or 28. *(Doom does not support development builds like 29+ or any version ending in .50 or .9x)*

### Expected behavior

When I do "SPC /" in any deeper than a root directory of the project and do "C-c C-l" to "embark-collect" results after that, I can't jump to any file from that result buffer cause a path that command trying to reach is broken somehow. 

### Current behavior

When I try to reach a file (Hit enter on the link) from embark-collect buffer results it tries to reach a not-existing path.

### Steps to reproduce

Open known projects with "SPC p p"
Hit "." and go to any directory inside
Hit "/" and do some search
Hit "C-c C-l" to export results with embark-collect
Hit (press Enter) on any file-link from this results
Any file you've been trying to open gonna be not-existed

### System Information

https://pastebin.com/mXLgXP8Q

**Comment by ClassAteam** [comment]
> Did this use to work? There is a recent embark change that might have caused this, I'll look in to it later.

Yes, it was.
Thank you !

**Comment by rymndhng** [comment]
I've noticed this as well. 

It appears to be fixed upstream in `embark`, possibly by this: https://github.com/oantolin/embark/issues/616.

Workaround: I've been able to bypass this bug by pinning embark to a newer version:

```
(package! embark :pin "8829e6c8cf0c34f67bfca1c494c809e196649ef1")
(package! embark-consult :pin "8889f5b09157a42bd975e739a9fcc7ffd0931cd8")
```

**Comment by ClassAteam** [comment]
> I've noticed this as well.
> 
> It appears to be fixed upstream in `embark`, possibly by this: [oantolin/embark#616](https://github.com/oantolin/embark/issues/616).
> 
> Workaround: I've been able to bypass this bug by pinning embark to a newer version:
> 
> ```
> (package! embark :pin "8829e6c8cf0c34f67bfca1c494c809e196649ef1")
> (package! embark-consult :pin "8889f5b09157a42bd975e739a9fcc7ffd0931cd8")
> ```

 Don't work for me though.

**Comment by dschrempf** [comment]
I can confirm the bug, I didn't try the workaround though.

**Comment by davidhaley** [comment]
I'm experiencing the same issue (even after trying the pinned versions).

**Comment by dschrempf** [comment]
One "workaround" is to use `embark-export` instead of `embark-collect`. It sucks a bit though, I know.

**Comment by rymndhng** [comment]
It seems to be fixed in the latest commit: 07fca78.

**Comment by ClassAteam** [comment]
Seems like OK now. Thank you.
